bentoml/saved_bundle/templates.py

```diff
@@ -75,35 +75,39 @@
 ARG EXTRA_PIP_INSTALL_ARGS=
 ENV EXTRA_PIP_INSTALL_ARGS $EXTRA_PIP_INSTALL_ARGS
 
+ARG UID=1034
+ARG GID=1034
+RUN groupadd -g $GID -o bentoml && useradd -m -u $UID -g $GID -o -r bentoml
+USER bentoml
+WORKDIR /home/bentoml
+ENV BENTOML_HOME=/home/bentoml/var/
+
 # copy over files needed for init script
-COPY environment.yml requirements.txt setup.sh* bentoml-init.sh python_version* /bento/
-WORKDIR /bento
+COPY --chown=bentoml:bentoml environment.yml requirements.txt setup.sh* bentoml-init.sh python_version* ./
 
 # copy over entrypoint scripts
-COPY docker-entrypoint.sh /usr/local/bin/
+COPY --chown=bentoml:bentoml docker-entrypoint.sh ./
```

Comment:
merge this line with L86?

Reply:
I chose to split the steps in `bentoml-init.sh` following the [dockerfile_best-practices](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#leverage-build-cache)